### DISCUSSION TITLE
BFE https 1个进程（1个product）  是否支持   配置2个 域名，2个数字证书？

### DISCUSSION THREAD

**Discussion by cherishman2005** [bfenetworks/bfe#1164] [Q&A (问题和回答)]
BFE https 1个进程（1个product）  是否支持   配置2个 域名，2个数字证书？

**Comment by Tovi163** [comment]
> BFE https 1个进程（1个product） 是否支持 配置2个 域名，2个数字证书？

理论上，一个product 可以支持配置多个证书；当然你可以将两个域名合并到一个证书中
